app/constants.py

```diff
@@ -143,6 +181,8 @@
 EMAIL_TYPE = "email"
 LETTER_TYPE = "letter"
 BROADCAST_TYPE = "broadcast"
+ALL_TYPE = "all"
+ALL_NOTIFICATION_TYPES = [SMS_TYPE, EMAIL_TYPE, LETTER_TYPE]
```

Comment:
The `NOTIFICATION_TYPES` constant 2 lines down covers all three notification types so can be used instead of adding a new constant